include/grpcpp/impl/codegen/callback_common.h

```diff
@@ -101,10 +101,11 @@ class CallbackWithStatusTag
     GPR_CODEGEN_ASSERT(ignored == ops_);
 
     // Last use of func_ or status_, so ok to move them out
-    CatchingCallback(std::move(func_), std::move(status_));
-
+    auto func = std::move(func_);
```

Comment:
Sorry, I didn't see your review comments when I merged. The temp variables allow us to reset the fields of the call arena before calling the callback. This ended up being an issue in the server callbacks that was manifesting as a double free since we were emptying out the call op set there as well as here.